### DISCUSSION TITLE
Saving and reloading workspaces and panes

### DISCUSSION THREAD

**Discussion by ssijak** [wezterm/wezterm#6067] [Q&A]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

Hey, is there a native solution to save and reload all workspaces and tabs and panels inside of them? I've seen a github repo for this that seems abandoned and did not work as expected. I was wondering if there is a native wezterm way to do it? I've seen a cli command to list all workspaces as a json file, but is there a way to load it back?

**Comment by pyrho** [comment]
Checkout https://github.com/MLFlexer/resurrect.wezterm seems to work fine!
